### ISSUE TITLE
Problem with dragging features

### ISSUE THREAD

**Issue by vince1995** [alex3165/react-mapbox-gl#692]
Hey, 

I have some `Feature`s wich are draggable. When I start to drag a marker and drag it fast over another marker it jumps back to its first position. It neither doesn't call `onDragEnd`.

Here's an gif:

![test 1](https://user-images.githubusercontent.com/15120985/52479739-eb614a80-2ba9-11e9-8f58-a9b8c851a514.gif)

**Comment by vince1995** [comment]
Ok, I've worked around this problem by using `Marker`s instead of `Layer` and `Feature`s.

**Comment by cristian-calugar** [comment]
Is Marker draggable? Based on documentation only Feature component has draggable prop.

**Comment by vince1995** [comment]
No, I've added mouse events and calculate the position by myself.
